Makefile

```
@@ -92,10 +94,18 @@ install-identity-scripts: install-datadir
 	install -m 755 -d $(prefix)$(datadir)/mender/identity
 	install -m 755 $(IDENTITY_SCRIPTS) $(prefix)$(datadir)/mender/identity/
 
-install-inventory-scripts: install-datadir
+install-inventory-scripts: install-datadir install-inventory-network-scripts
```

This should be:
```suggestion
install-inventory-scripts: install-local-inventory-scripts install-inventory-network-scripts
```
(sic)

---

Hmm, yeah. Good idea. Then `install-inventory-scripts` is just a shim target, tying together the `local` and `network` targets.